eisenhower-matrix/src/main/webapp/index.html

```diff
@@ -16,10 +16,6 @@
   <body>
     <nav id="header" class="navbar navbar-expand-lg navbar-light d-flex justify-content-between">
         <h1>Eisenhower Matrix Scheduler</h1>
-      <div  class="flex-row-reverse" id="authentification">
-          <button id="logout" class="btn btn-default " >Logout</button>
-          <button id="login" class="btn btn-default ">Login</button>
-      </div>
```

Comment:
Why don't we need the login/logout buttons?

Reply:
I'll re add these; for some reason I thought you wanted them gone in favor of GCal login

Reply:
@deeptivaidy rn they aren't linking to any function....

Reply:
Oh I'm linking them in script.js since it fit easier with the tutorial I found, really the login and logout functions in script.js aren't used